src/Agent.Plugins/GitCliManager.cs

```diff
@@ -441,6 +441,26 @@ public async Task<bool> GitConfigExist(AgentTaskPluginExecutionContext context,
             return exitcode == 0;
         }
 
+        /// <summary>
+        /// Verify if git config contains config key with some regex pattern
+        /// git config --get-regexp <configKeyPattern>
+        /// </summary>
+        /// <param name="context"></param>
+        /// <param name="repositoryPath"></param>
+        /// <param name="configKeyPattern"></param>
+        /// <returns></returns>
```

Comment:
In `<param>` it is better to add a description of the parameters,
and in `<returns>` add a description of the returned value.
Otherwise, it is better to remove the `<param>` and `<returns>` altogether.